docker/secondary-analysis-python/Dockerfile

```diff
@@ -5,3 +5,9 @@ LABEL software="Python"
 LABEL description="Python with some useful libraries"
 
 RUN pip install requests
+
+RUN mkdir /tools
+
+COPY tools /tools
```

Comment:
Could this be replaced by: 
```
RUN git clone https://github.com/HumanCellAtlas/skylab.git && \
    pip install -e skylab/docker/secondary-analysis-python/tools/submit
```

Reply:
The development process I want to enable is that I make changes to submit.py in my local copy of the repo, then I use the test.sh script to build and run the docker image and run unit tests in it. So for that use case it's important to have the docker image built from my local copy, so that my changes are reflected in it and tested.

I'm assuming that anyone else who has the secondary-analysis-python Dockerfile will have cloned the repo so that pip installing from tools/submit will work.

Does that make sense?

Reply:
Makes perfect sense, thanks!